support/azure/azure-kubernetes/availability-performance/container-image-pull-performance.md

```diff
@@ -16,7 +16,9 @@ This article provides methods to improve container image pull performance in Azu
 
 There are two types of container image pulls: serialized and parallel image pulls.
 
-In AKS versions earlier than 1.31, AKS enables serialized image pulls by default. Typically, serialized image pulls should be less performant than parallel image pulls. Starting from AKS version 1.31 preview, AKS pulls container images in parallel by default. This change is intended to improve container image pull performance, especially when you deal with large or numerous container images. However, you might still experience decreased performance compared to using serialized image pulls.
+In AKS versions earlier than 1.31, AKS enables serialized image pulls by default. Starting from AKS version 1.31 preview, AKS pulls container images in parallel by default. Typically, serialized image pulls should be less performant than parallel image pulls, particularly when attempting to pull large or numerous container images.
+
+However, you might still experience decreased performance with parallel compared to using serial image pulls. There is possibility to throttle the disk when trying to pull large or numerous images in parallel fashion, especially with unoptimized disk and VM resources. In this situation, you may notice that the time to complete the pull of the first images is faster with serial, but the time to pull all images is faster with parallel. Depending on your workload needs, consider following the steps below to scale your disk and VM resources or toggle the image pull type.
```

Comment:
However, you might still experience decreased performance by using parallel image pulls compared to using serial image pulls. Trying to pull large or numerous images in a parallel manner might throttle the disk. This is especially true when you use unoptimized disk and VM resources. In this situation, you might notice that the time to pull the first images is faster in a serial setup, but the time to pull all images is faster in a parallel setup. Depending on your workload needs, you should consider using the following steps to scale your disk and VM resources or toggle the image pull type.